src/Css.elm

```diff
@@ -1417,6 +1443,18 @@ right =
     Value "right"
 
 
+{-| -}
+top : Value { provides | top : Supported }
+top =
+    Value "top"
```

Comment:
I think these will need to be `top_`, `bottom_`, `left_`, and `right_` because there are already functions with those names (e.g. `top (px 20)` used with absolute positioning.) 😅 

I'd change both the value names as well as making these `top_ : Supported` so the docs and error messages and value names are all consistent!

I'd also add docs to them which mention the other ones - e.g. the docs for `top_` could say "The `top` value used in color stops. This value is called `top_` instead of `top` because [`top` is a function](#top) already."

Reply:
I'll change left_ and right_ too, since they'll have the same problem and they're already `Value`s